util_runner.go

```diff
@@ -168,3 +179,20 @@ func createPidFile(path string, process *libcontainer.Process) error {
 	}
 	return os.Rename(tmpName, path)
 }
+
+func adjustOomScore(process *libcontainer.Process) error {
+	pid, err := process.Pid()
+	if err != nil {
+		return err
+	}
+	oomScoreAdjPath := path.Join("/proc/", strconv.Itoa(pid), "oom_score_adj")
```

Comment:
Since this is host specific, let's use `filepath.Join` instead, we already have it imported as well :).